layouts/_default/list.algolia.json

```diff
@@ -35,6 +35,9 @@
         {{ end }}
     {{ end }}
 
+    {{- /* api pages are controlled by layout and not markdown, so we need to index them separately */ -}}
+    {{ partial "algolia/api-pages-full-index.json" (dict "context" $hugo_context) }}
+
```

Comment:
here we are ignoring all api pages in the main loop and later iterating over all the `api/latest` pages only and adding to the main `algoliaindex` object.   i think we can make this process more efficient by including the api `v1` and `v2` pages in with the main loop and build them up as `api/latest` entries in the index.